docs/standard/simd.md

```diff
@@ -80,7 +80,7 @@ double[] SimdVectorProd(double[] left, double[] right)
     var offset = Vector<double>.Count;
     double[] result = new double[left.Length];
     int i = 0;
-    for (i = 0; i < left.Length; i += offset)
+    for (i = 0; i + offset < left.Length; i += offset)
```

Comment:
The change here is correct, but its also not the "best" way to do all this and misses another case where `right.Length < left.Length`.

If we were to update the sample to avoid bugs, etc; I'd probably write this as the below. This switches to using `Sum` which I believe is simpler to understand/validate and which is always accelerated (multiplication is generally accelerated but there are edge cases where it isn't). It validates the inputs and tracks the length in a fashion that is more extensible for the code to handle other interesting scenarios if that's ever desired.

---------------------------

> The example below demonstrates how to calculate the element-wise sum of two arrays using <xref:System.Numerics.Vector%601>.
```csharp
double[] Sum(double[] left, double[] right)
{
    if (left is null)
    {
        throw new ArgumentNullException(nameof(left));
    }

    if (right is null)
    {
        throw new ArgumentNullException(nameof(right));
    }

    if (left.Length != right.Length)
    {
        throw new ArgumentException($"{nameof(left)} and {nameof(right)} are not the same length");
    }

    int length = left.Length;
    double[] result = new double[length];

    // Get the number of elements that can't be processed in the vector
    // NOTE: Vector<T>.Count is a JIT time constant and will get optimized accordingly
    int remaining = length % Vector<double>.Count;

    for (int i = 0; i < length - remaining; i += Vector<double>.Count)
    {
        var v1 = new Vector<double>(left, i);
        var v2 = new Vector<double>(right, i);
        (v1 * v2).CopyTo(result, i);
    }

    for (int i = length - remaining; i < length; i++)
    {
        result[i] = left[i] * right[i];
    }

    return result;
}
```

Reply:
@tannergooding isn't your example still doing multiplication instead of computing the sum?

Reply:
Yes, fixed.